jupyterhub/spawner.py

```diff
@@ -174,12 +263,16 @@ def _deprecate_percent_u(self, proposal):
             v = v.replace('%U', '{username}')
             self.log.warning("Converting %r to %r", proposal['value'], v)
         return v
-    
-    disable_user_config = Bool(False,
-        help="""Disable per-user configuration of single-user servers.
-        
-        This prevents any config in users' $HOME directories
-        from having an effect on their server.
+
+    disable_user_config = Bool(
+        False,
+        help="""
+        Disable per-user configuration of single-user servers.
+
+        This prevents any config in users' $HOME directories from having an effect on their server.
```

Comment:
... when starting their single-user server.  (@minrk, please verify)

I'm not sure that the sentence of circumventing belongs here. Thoughts?